website/www/site/content/en/get-started/downloads.md

```diff
@@ -87,6 +87,13 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.23.0 (2020-07-16)
```

Comment:
Don't forget to update this if release is finalized on a different date.

Reply:
Done, thanks.